### PR TITLE
Add recipe for rjchallis/assembly-stats

### PR DESCRIPTION
**Pull request by hexylena**
Adds a recipe for https://github.com/rjchallis/assembly-stats which produces quite nice graphs for comparing genome assemblies.

This tool has two perl scripts, as well as some html/css/js that need to be loaded in order to visualise. I've followed the jbrowse recipe for some inspiration there, but I do not feel sure about the solution here, input is welcome.

The html file included needs to be copied to do the visualisation, and the two json files produced by the perl script actually need to be merged, which isn't part of the tooling at all. 

@bgruening per discussion in gga channel with @abretaud 

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/rjchallis-assembly-stats/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+test:
+  commands:
+    - perl -e 'print ">test\n"; print "A" x 1000' | asm2stats.pl
+    - perl -e 'print ">test\n"; print "A" x 1000' | asm2stats.minmaxgc.pl
+
+about:
+  home: "https://github.com/rjchallis/assembly-stats"
+  summary: Assembly metric visualisations to facilitate rapid assessment and comparison of assembly quality.
+  license: MIT License
```
Comment (bgruening) [inline recipes/rjchallis-assembly-stats/meta.yaml:30]: can you please use here SPDX IDs: https://spdx.org/licenses/MIT.html

Comment (hexylena) [reply recipes/rjchallis-assembly-stats/meta.yaml:30]: did not know what went there and grepped for examples, I guess they were misleading, thanks!

**Comment by abretaud** [comment]
Tested and approved locally :+1: 
Ready to merge @hexylena?

**Comment by hexylena** [comment]
Ready from my side! Thanks for fixing those sed's @abretaud !

**Review by abretaud — APPROVED** [review]

**Comment by abretaud** [comment]
@BiocondaBot please merge

**Pull request merged**